tools/GraphClust/AlignCluster/align_cluster.xml

```diff
@@ -0,0 +1,230 @@
+<tool id="align_cluster" name="graphclust_align_cluster" version="0.1" >
+  <requirements>
+      <requirement type="package" version="0.6.0">graphclust-wrappers</requirement>
+      <requirement type="package" version='0.5'>perl-array-utils</requirement>
+      <requirement type="package" version='0.18.1'>scikit-learn</requirement>
+      <requirement type="package" version='1.8.10'>locarna</requirement>
+      <requirement type="package" version='2.1'>rnaz</requirement>
+      <requirement type="package" version="1.1.2">infernal</requirement>
+      <requirement type="package" version='2.2.10'>viennarna</requirement>
+      <requirement type="package" version='1.3.30'>graphicsmagick</requirement>
+      <requirement type="package" version='0.6.1'>rscape</requirement>
+      <requirement type="package" version='6.0'>unzip</requirement>
+      <requirement type="package" version='0.1'>evofold2</requirement>
+      <requirement type="package" version='1.70'>biopython</requirement>
+      <requirement type="package" version='0.23.0'>pandas</requirement>
+  </requirements>
+  <command detect_errors="aggressive">
+  <![CDATA[
+      #set $alignment_clustal = 'cluster.aln'
+      resDir="./RESULTS/" &&   
```

Comment:
you are not using this dir, isn't it?

Reply:
It's passed to the perl script at line 24.

Reply:
no its not, is it? `resDir` is not used? You are using everywhere ` ./RESULTS/`

Reply:
True! fixed.